src/components/Modal.svelte

```diff
@@ -140,4 +145,4 @@
       </div>
     {/if}
   </section>
-</aside>
+</aside>
```

Comment:
Nit: Missing newline (please check everywhere and maybe add it to the lint checks).